deepchem/models/IRV.py

```diff
@@ -5,12 +5,26 @@
 from deepchem.models.losses import SigmoidCrossEntropy
 from tensorflow.keras.layers import Input, Layer, Activation, Concatenate, Lambda
 
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
 
-class IRVLayer(Layer):
```

Comment:
Don't modify existing tensorflow implementation. Instead add a new file to deepchem/models/torch_models (irv.py).